responsibleai_vision/responsibleai_vision/managers/explainer_manager.py

```diff
@@ -577,11 +585,13 @@ def _is_object_detection_task(self):
         return self._task_type == ModelTask.OBJECT_DETECTION
 
     def _get_fail_str(self):
-        fail = Image.new('RGB', (100, 100))
+        fail = Image.new('RGB', (250, 250))
```

Comment:
Why is this changed/needed?

Reply:
Because the text is way longer than 100 and it's just getting cut off. That makes this entire method pointless. With 250 I found it fits comfortably without wasting much space.

Reply:
I wasn't sure if it's required to be square so I kept that.